src/routes/safes/entities/safe-info.entity.ts

```diff
@@ -20,13 +20,13 @@ export class SafeState {
   readonly owners: AddressInfo[];
   @ApiProperty()
   readonly implementation: AddressInfo;
-  @ApiPropertyOptional()
+  @ApiPropertyOptional({ type: AddressInfo, nullable: true })
   readonly modules: AddressInfo[] | null;
-  @ApiPropertyOptional()
+  @ApiPropertyOptional({ type: AddressInfo, nullable: true })
   readonly fallbackHandler: AddressInfo | null;
-  @ApiPropertyOptional()
+  @ApiPropertyOptional({ type: AddressInfo, nullable: true })
```

Comment:
nitpick: I would create a separate commit for this and not squash it (if done in the same PR) or a separate PR. Consider atomic changes that we should revert if required.

Reply:
I agree, good point! I've left all the changes directly related to this issue in [2a001a3](https://github.com/safe-global/safe-client-gateway/pull/764/commits/2a001a35688205a3e7712d909525a01b25c04778), and I've created another PR for the rest: https://github.com/safe-global/safe-client-gateway/pull/765